File: DesktopClient/src/sample/document/Message.java
package sample.document;

public class Message {
    private String sender;
    private String song;


    public Message(String sender, String song){
        this.sender=sender;
        this.song=song;
    }
    public String getMessage(){
        return "From: "+this.sender+". Hey you need to listen this song: "+this.song;
    }

}

File: OdyServer/Objects/message.cpp
#include "message.h"


Message::Message(string sender, string receiver, string song)
{
    this->sender=sender;
    this->receiver=receiver;
    this->song=song;

}

string Message::getSong()
{
    return this->song;
}

string Message::getSender()
{
    return this->sender;
}

string Message::getReceiver()
{
    return this->receiver;
}
